js/components/uswds_checkbox_widget.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+const USWDSCheckboxWidget = function(props) {
```

Comment:
Would you change this to `function USWDSCheckboxWidget({ label, value }) {`? It helps with debugging to have named functions.

Reply:
Ah, sure! Should there be a test for this? I wasn't entirely sure.